RHDEVDOCS 6367 add explanation about pod templates to Pipelines 1.16 RN (#88657)
openshift/openshift-docs: modules/op-release-notes-1-16.adoc

* With this update, the metric name for the `EventListener` object for pipelines triggers that counts received events is changed from `eventlistener_event_count` to `eventlistener_event_received_count`.

> * Before this update, when using {pac}, the pipeline run executed correctly if you specified the `podTemplate` parameters for a pipeline run using one of the following  wrong ways for the API version:

Reviewer: ```suggestion
* Before this update, when using {pac}, the pipeline run executed correctly if you specified the `podTemplate` parameters for a pipeline run using one of the following wrong ways for the API version:
```

+
** For the `v1beta1` API, in the `taskRunTemplate.podTemplate` spec
** For the `v1` API, in the `podTemplate` spec
+

> With this update, with one of these wrong specifications, the `podTemplate` parameters are disregarded.

Reviewer: suggestion:
```suggestion
With this update, when you make either of the incorrect specifications, the `podTemplate` parameters are disregarded.
```
